src/SprykerEco/Zed/ComputopApi/Business/Request/ExpressCheckout/AbstractPayPalExpressRequest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\ComputopApi\Business\Request\ExpressCheckout;
+
+use Spryker\Shared\Kernel\Transfer\TransferInterface;
+use SprykerEco\Zed\ComputopApi\Business\Adapter\AdapterInterface;
+use SprykerEco\Zed\ComputopApi\Business\Converter\ConverterInterface;
+use SprykerEco\Zed\ComputopApi\Business\Mapper\ExpressCheckout\PayPalExpressMapperInterface;
+
+abstract class AbstractPayPalExpressRequest
+{
+    /**
+     * @var \SprykerEco\Zed\ComputopApi\Business\Adapter\AdapterInterface
+     */
+    protected $adapter;
+
+    /**
+     * @var \SprykerEco\Zed\ComputopApi\Business\Converter\ConverterInterface
+     */
+    protected $converter;
+
+    /**
+     * @var \SprykerEco\Zed\ComputopApi\Business\Mapper\ExpressCheckout\PayPalExpressMapperInterface
+     */
+    protected $payPalExpressMapper;
```

Comment:
same as above
looks like specific child dependency

Reply:
this property is used in both child classes